dedupe/clustering.py

```diff
@@ -74,6 +74,9 @@ def _connected_components(edgelist: numpy.ndarray,
 
         if n_components > max_components:
             min_score = numpy.min(sub_graph['score'])
+            if min_score == 0:
```

Comment:
After https://github.com/dedupeio/dedupe/pull/809 was merged, I think we shouldn't see `0` scores at clustering stage. Does your code have those changes?